src/sentry/static/sentry/app/components/hovercard.jsx

```diff
@@ -202,7 +209,10 @@ const StyledHovercard = styled('div')`
   background: #fff;
   background-clip: padding-box;
   box-shadow: 0 0 35px 0 rgba(67, 62, 75, 0.2);
-  width: 295px;
+  ${p => {
+    return p.shrinkToFit ? 'max-width: 295px;' : 'width: 295px;';
```

Comment:
Because you only have a single statement you could drop the return and braces :golf:

Reply:
Yeah, the linter is failing because of that (and because of a missing newline somewhere in the imports)... and yet in VSCode the linter is perfectly happy. Aren't they supposed to be working off of the same set of rules?

I tried running `yarn lint --fix`, but nothing changed. I know what it wants in the case you mentioned above (so I can fix that manually), but not in the other case. Suggestions on how to proceed?

Reply:
(Also, running `yarn lint` locally lead to 153 _other_ things it's mad about, in files I didn't touch. Not sure why those didn't fail on Travis, and also not sure what to do about them.

Reply:
I don't know how to setup vscode, but you can run eslint for a single file using `node_modules/.bin/eslint --fix <file>`